tests/api/v1/endpoints/test_health_endpoints.py

```diff
@@ -0,0 +1,7 @@
+from starlette.testclient import TestClient
+
+
+def test_health(api_client: TestClient) -> None:
+    url = "http://0.0.0.0:8080/health"
```

Comment:
Rather than hardcoding this here, it might be a good opportunity to add `/health` to `urn_registry.py`  so it can be reused to both decorate the view and used as the URL in this test.

Reply:
Makes sense to me. I added it to the registry and updated references to the URL.